Libraries/LibWeb/IndexedDB/IDBKeyRange.h

```diff
@@ -18,11 +23,41 @@ class IDBKeyRange : public Bindings::PlatformObject {
 
 public:
     virtual ~IDBKeyRange() override;
-    [[nodiscard]] static GC::Ref<IDBKeyRange> create(JS::Realm&);
+    [[nodiscard]] static GC::Ref<IDBKeyRange> create(JS::Realm&, GC::Ptr<Key> lower_bound, GC::Ptr<Key> upper_bound, bool lower_open, bool upper_open);
+
+    [[nodiscard]] JS::Value lower() const;
+    [[nodiscard]] JS::Value upper() const;
+    [[nodiscard]] bool lower_open() const { return m_lower_open; }
+    [[nodiscard]] bool upper_open() const { return m_upper_open; }
+
+    [[nodiscard]] static WebIDL::ExceptionOr<GC::Ref<IDBKeyRange>> only(JS::VM&, JS::Value);
+    [[nodiscard]] static WebIDL::ExceptionOr<GC::Ref<IDBKeyRange>> lower_bound(JS::VM&, JS::Value, bool);
+    [[nodiscard]] static WebIDL::ExceptionOr<GC::Ref<IDBKeyRange>> upper_bound(JS::VM&, JS::Value, bool);
+    [[nodiscard]] static WebIDL::ExceptionOr<GC::Ref<IDBKeyRange>> bound(JS::VM&, JS::Value, JS::Value, bool, bool);
+    [[nodiscard]] WebIDL::ExceptionOr<bool> includes(JS::Value);
```

Comment:
Isn't `WebIDL::ExceptionOr` already marked with `[[nodiscard]]`?

Reply:
Thats true 😅 Removed the extra